core/trino-spi/pom.xml

```diff
@@ -254,6 +254,77 @@
                                     <old>interface io.trino.spi.ptf.TableFunctionDataProcessor</old>
                                     <justification>Relocate table function infrastructure</justification>
                                 </item>
+                                <item>
+                                    <ignore>true</ignore>
```

Comment:
We actually don't need `<ignore>true</ignore>` since that is set at the top level